docs/advanced/command-line/files/sources.md

```diff
@@ -0,0 +1,36 @@
+# Sources
+
+The file describing how to access the sources, passed as an argument to the -s command line option
+
+## Format of the file
+We have provided an example of how the contents of the file should be structured in **sources.csv** in the Manual Books example.
+
+The csv file is expected to contain exactly the following columns:
+
+1. uri. A jdbc connection string to an external database, as described in [Adding Data Uri](../../../../guides/adding-data/local-DB/). If a uri
+is not specified here, then the source entry should be defined in column data.
+
+2. tables. Optionally, in case a uri is provided, a comma-separated list of the specific tables in the database schema to consider in the wrangling process. If none is specified, all tables will be considered.
+
+3. description. Optionally, a description of the resource.
```

Comment:
i think it makes more sense to read (optional) rather than Optionally,

Reply:
Done!